plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiPlugin.java

```diff
@@ -45,6 +46,19 @@ public void testCreateTestingConnector()
                 .shutdown();
     }
 
+    @Test
```

Comment:
nit: you could have added that in previous commit; just marked as ignored with comment. 
And here just fix and unignore.

Reply:
agreed. or i could add it as a failing test, showing failure, and then showing the fix.
i think it doesn't matter for the long-term, and don't want to overload the CI queue, so will merge as is